inst/tutorials/statistical_models_ws/statistical_models_ws.Rmd

```diff
@@ -0,0 +1,1918 @@
+```
+longevity ~ activity
+```
+The formula notation can get more complex, such as including additional explanatory/independent variables or interaction terms. We will introduce these as we attempt more complex analyses later on.
+
+#### ANOVA in R
+We want to know if sexual activity has an effect on population mean longevity of male fruit flies. 
+The `aov()` function can be used to create a model that answers this question. 
+As stated above, the first argument to this function is the formula. 
+The second argument is the name of the data frame variable, in our case  `fly_2_groups`.
+
+Create an ANOVA model object called `fly_2_groups_model`:
+
+```{r two-group, exercise = TRUE}
+# The response variable is on the left and explanatory variable on the right
+fly_2_groups_model <- aov(<response_column> ~ <explanatory_column>,
+                          data = <variable_name>)
+```
+
+```{r two-group-solution}
+fly_2_groups_model <- aov(longevity ~ activity, data = fly_2_groups)
+```
+
+The `aov()` function returns a "model" object. You can examine the results of the model with the `summary()` function:
+
+```{r two-group-sum, exercise = TRUE}
+summary(<model_object>)
+```
+
+```{r two-group-sum-solution, excercise.setup = "setup-two-group-sum"}
+summary(<fly_2_groups_model>)
+```
+
+
+So, what does this output mean? The most important result in regards to rejecting or failing to reject our null hypothesis is the right-most number, the p-value (`Pr(>F)`). In this simple one-way ANOVA, we have a single p-value which has a very small value. Given that this is much smaller then the commonly used threshold for rejecting the null hypothesis, p < 0.05, we can reject our null hypothesis that sexual activity has no effect on the population mean longevity of male fruit flies, and accept the alternative hypothesis that sexual activity **does** have an effect on population mean longevity of male fruit flies. 
+
+#### **Exercise 1: 1-way ANOVA**
+Using ANOVA, test if fruit fly longevity is affected by size (as measured by thorax length).
+
+  * What are your null and alternate hypotheses?
+  * What can you conclude from these results?
+
+Create a model variable called `exercise1` and use the `summary()` function to examine the output.
+
+**Note: Hints will be provided for exercises but no solution. Try to figure it out!**
+```{r ex1, exercise = TRUE}
+<model_object> <- aov(<response_column> ~ <explanatory_column>,
+                      data = <variable_name>)
+<function_for_examining_output>(<model_object>)
+```
+
+```{r ex1-hint-1}
+# The response column is thorax and the exploratory column is thorax
+```
+
+```{r ex1-hint-2}
+# Name the model object exercise1!
+```
+
+```{r ex1-hint-3}
+# To examine the output of your model, use the summary() function!
+```
```

Comment:
````suggestion
::: exercise
**1-way ANOVA**: Using ANOVA, test if fruit fly longevity is affected by size (as measured by thorax length).

  * What are your null and alternate hypotheses?
  * What can you conclude from these results?

Create a model variable called `exercise1` and use the `summary()` function to examine the output.

**Note: Hints will be provided for exercises but no solution. Try to figure it out!**
```{r ex1, exercise = TRUE}
<model_object> <- aov(<response_column> ~ <explanatory_column>,
                      data = <variable_name>)
<function_for_examining_output>(<model_object>)
```

```{r ex1-hint-1}
# The response variable is longevity, and the explanatory variable is thorax.
```

```{r ex1-hint-2}
# Name the model object exercise1!
```

```{r ex1-hint-3}
# To examine the output of your model, use the summary() function!
```
:::

````
I just pushed the necessary `style.css` to use the exercise `<div>` as demonstrated here.

The code chunk above does not work. Please fix.